CHANGELOG.md

```diff
@@ -4,6 +4,49 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
+### Framework
+
+- [Optimization] Update default rust toolchain. [@yangby]
+- [Optimization] Update cita-sdk-js version. [@kaikai]
+- [Fix] Reorganize toml path. [@WPF]
+- [Feature] Config the log module. [@kaiyu]
```

Comment:
Is better change to `Make CITA's log output mode can be configured as stdout or log-file` ?

Reply:
nice. I will take part of it:

> Log output mode can be configured as stdout or log-file